### PR TITLE
updated download-dialog for converse v7

### PR DESCRIPTION
**Pull request by ChaosKid42**
This PR updates the download-dialog plugin to be compatible with converse v7. Should probably not be merged before converse v7 is out.

### CODE REVIEW

File: packages/download-dialog/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@converse-plugins/download-dialog",
+  "version": "0.0.1",
+  "keywords": [
+    "converse.js",
+    "plugin"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
```
Comment (jcbrand) [inline packages/download-dialog/package.json:10]: I wanted to mention that you'll need to set this, but I see you've done so already.

**Comment by ChaosKid42** [comment]
@jcbrand @deleolajide Updated this PR in order to use lit-html instead of lodash.
Do you think we shall merge this now?

**Comment by ChaosKid42** [comment]
@deleolajide Rebased my PR to current HEAD so it can be merged when converse v7 comes out.

**Comment by ChaosKid42** [comment]
@deleolajide I'd like to merge this. What do you think?

**Comment by deleolajide** [comment]
> I'd like to merge this. What do you think?

Sure thing. At the moment, the test index.html page is pointing to Converse 6.0.1. I was thinking we should point it at 7.0

**Pull request merged**